examples/labelelide-example.ts

```diff
@@ -233,7 +254,7 @@ and warm within me, that it might be the mirror of my soul, as my soul is the mi
             this._labelCustom.ellipsis = '.'.repeat(Math.floor(Math.sin(performance.now() * 0.001) * 4.0) + 5);
 
             this.invalidate();
-        }, 1000.0 / 60.0);
+        }, 1000.0 / 60.0) as unknown as number;
```

Comment:
I find this conversion weird. It is to avoid `Type 'Timer'  is not assignable to type 'number'`?
https://github.com/TypeStrong/atom-typescript/issues/1053

https://stackoverflow.com/questions/53189729/typescript-error-setinterval-type-timer-is-not-assignable-to-type-number

http://evanshortiss.com/development/nodejs/typescript/2016/11/16/timers-in-typescript.html

Reply:
On my machine, the return type of `setInterval` was even weirder: `NodeJS::TimeOut`.

Reply:
is it possible to apply some solutions suggested by the linked sources? like, `window.setInterval`, `global.setInterval` ?